include/openthread-coap.h

```diff
@@ -224,6 +239,31 @@ ThreadError otCoapHeaderAppendOption(otCoapHeader *aHeader, const otCoapOption *
 ThreadError otCoapHeaderAppendUriPathOptions(otCoapHeader *aHeader, const char *aUriPath);
 
 /**
+ * This function appends a Max-Age option.
+ *
+ * @param[inout]  aHeader   A pointer to the CoAP header.
+ * @param[in]     aMaxAge   The Max-Age value.
+ *
+ * @retval kThreadError_None         Successfully appended the option.
+ * @retval kThreadError_InvalidArgs  The option type is not equal or greater than the last option type.
+ * @retval kThreadError_NoBufs       The option length exceeds the buffer size.
+ */
+ThreadError otCoapHeaderAppendMaxAgeOption(otCoapHeader *aHeader, uint32_t aMaxAge);
+
+/**
+ * This function appends a single Uri-Query option.
+ *
+ * @param[inout]  aHeader   A pointer to the CoAP header.
+ * @param[in]     aUriQuery A pointer to NULL-terminated string, which should contain
+ *  a single key=value pair.
```

Comment:
I didn't want it to be on the same column as @param, so just let the IDE to auto-align it. I'll make it one line, seeing as there are longer lines in that file.